dockerfile/anaconda-iso-creator/pkexec-command.patch

```diff
@@ -0,0 +1,40 @@
+From 963d74a7df9502112df4929ca48a5fb7c0910dc8 Mon Sep 17 00:00:00 2001
+From: Adam Kankovsky <adamkankovsky@gmail.com>
+Date: Thu, 3 Oct 2024 09:52:26 +0200
+Subject: [PATCH] Update cleanup for pkexec command
+
+---
+ share/templates.d/99-generic/runtime-cleanup.tmpl     | 1 -
+ share/templates.d/99-generic/runtime-postinstall.tmpl | 4 ++++
+ 2 files changed, 4 insertions(+), 1 deletion(-)
+
+diff --git a/share/templates.d/99-generic/runtime-cleanup.tmpl b/share/templates.d/99-generic/runtime-cleanup.tmpl
```

Comment:
This needs to be fixed upstream Please. 

Create a PR and add a link here with FIXME to remove: https://github.com/weldr/lorax/blob/master/share/templates.d/99-generic/runtime-cleanup.tmpl